apps/src/StudioApp.js

```diff
@@ -26,6 +26,7 @@ var assetsApi = require('./clientApi').assets;
 var assetPrefix = require('./assetManagement/assetPrefix');
 var assetListStore = require('./assetManagement/assetListStore');
 var annotationList = require('./acemode/annotationList');
+var processMarkup = require('marked');
```

Comment:
Yeah, probably. I guess markdown is a particular flavor of markup http://stackoverflow.com/a/24044/2506748, which isn't at all confusing :-P